src/backend/base/langflow/api/v1/folders.py

```diff
@@ -127,6 +127,11 @@ def update_folder(
         ).first()
         if not existing_folder:
             raise HTTPException(status_code=404, detail="Folder not found")
+        if folder.name and folder.name != existing_folder.name:
+            existing_folder.name = folder.name
+            update(Folder).where(Folder.id == folder_id).values(name=folder.name)
```

Comment:
This is not doing anything. 
You need to pass the statement to the `session.exec`method.